markdown/ch3.markdown

```diff
@@ -1111,12 +1101,12 @@ def build_rag(labels, image):
     return g
 ```
 
-Here's a few reasons why we think this is a brilliant piece of code:
+Here's a few reasons this is a brilliant piece of code:
 
 - `ndi.generic_filter` iterates over array elements *with their neighbors*.
   (Use `numpy.ndindex` to simply iterate over array indices.)
 - We return "0.0" from the filter function because `generic_filter` requires
-  the filter function to return a float. However, we will ignore the filter
+  the filter function to return a float. However, we will ignore the filter's
```

Comment:
@stefanv look at latest changes. Looks like your browser cache wasn't showing them to you.